### PR TITLE
Yana simeonova

### PR DESCRIPTION
**Pull request by YanaSimeonova**

### CODE REVIEW

File: templates/sag-cc-all-layer-defs/template.yaml [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+                      '@alias': sag-db-oracle-103-and-lower   
+                        
+                CCE-LAYER-TYPES-DB-MYSQL-103:
+                  '@name': DB-MYSQL-103
+                  Description: MYSQL server database and sample product schemas for 10.3
+                  Type: DATABASE
+                  Scalable: 'false'
+                  Templates:
+                    Template:
+                      '@alias': sag-db-mysql-103
```
Comment (enstaneva) [inline templates/sag-cc-all-layer-defs/template.yaml]: is this for 10.3 only or again for lower, too?

File: templates/sag-db-mysql/template.yaml [hunk 2]
```diff
@@ -1,76 +1,158 @@
...
 templates:
   dbc:
-    products:  
-      DatabaseComponentConfigurator: 
-    fixes:     ${dbc.fixes}
+    products:  ${db.product.scripts}
+    fixes:     ${db.fixes}
     files:
-      # Using the fact that DCC picks up sl53* jars dynamically
+      # Coping MySQL JDBC driver as sl53* so DatabaseComponentConfigurator will pick it up dynamically
```
Comment (enstaneva) [inline templates/sag-db-mysql/template.yaml]: should it be copying?

File: templates/sag-cc-all-layer-defs/template.yaml (hunk 1 above)
Comment (YanaSimeonova) [reply templates/sag-cc-all-layer-defs/template.yaml]: The previous template was documented as 103+, I think the support started at that time.
But we can check again.

File: templates/sag-db-mysql/template.yaml (hunk 2 above)
Comment (YanaSimeonova) [reply templates/sag-db-mysql/template.yaml]: Thanks, coming in the next commit

**Pull request merged**